File: Question7/Question7.c
#include <stdbool.h>
#include <stdio.h>
#include <string.h>

bool solution(int n, int l, int transfers[][2])
{
    // Write solution here
    return false;
}

int main()
{
    int n, l;
    scanf("%d %d", &n, &l);

    int transfers[l][2];
    for (int i = 0; i < l; i++)
    {
        scanf("%d %d", &transfers[i][0], &transfers[i][1]);
    }

    bool answer = solution(n, l, transfers);
    printf("%s", answer ? "Ineligible" : "Eligible");

    return 0;
}

File: Question7/Question7.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;

class Solution {
    static bool solution(int n, int l, List<List<int>> transfers)
    {
        // Write solution here
        return false;
    }

    public static void Main(string[] args)
    {
        int[] firstLine = Array.ConvertAll(Console.ReadLine().Split(' '), containerTemp => Convert.ToInt32(containerTemp));

        int n = Convert.ToInt32(firstLine[0]);
        int l = firstLine[1];

        List<List<int>> transfers = new List<List<int>>();
        for (int i = 0; i < l; i++)
        {
            List<int> current = Array.ConvertAll(Console.ReadLine().Split(' '), containerTemp => Convert.ToInt32(containerTemp)).ToList();
            transfers.Add(current);
        }

        bool answer = solution(n, l, transfers);
        Console.WriteLine(answer ? "Ineligible" : "Eligible");
    }
}
